content/docs/authorization/v1.x/configuration/powermax/_index.md

```diff
@@ -142,7 +142,7 @@ Create the karavi-authorization-config secret using this command:
         # enabled: Always set to true
         enabled: true
         forceRemoveModule: true
-        configVersion: v2.12.0
+        configVersion: v2.11.0
```

Comment:
Both 1.x and 2.x pointing to the same config version?

Reply:
Last version for Reverse Proxy was v2.10.0, so it should be v2.11.0 for csipowermax-reverseproxy.